index.html

```diff
@@ -9,11 +9,87 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
-<body>
+
+<header>
+    
+    
+    <nav id="navbar" class="nav">
+        <img id="logo" src="img/karma-logo.svg" alt=# width="30" height="50" />  
+    
+        
+    <ul class="nav-list">
+      <li>
+        <a href="#welcome-section">meet Karama</a>
+      </li>
+      <li>
+        <a href="#projects">How it works</a>
+      </li>
+      <li>
+        <a href="#contact">stores</a>
+      </li>
+      <a href="#contact">blog</a>
+      </li>
+      <a href="#contact">Help</a>
+      </li>
+      <a href="#contact">Login</a>
+      </li>
+    </ul>
+  </nav>  
+     
+
+</header>
+  <body>
+  <section>
+     <h1>Introduction Karama </h1>
+     <p>  bring wifi with you,every where you go </p>
+     
+     <button >
+     <a href= "#"> learn more  </a>
+     </button>
```

Comment:
```suggestion
     <button >
       learn more
     </button>
```
It doesn't really make sense to have a link inside a button — it should be either a link or a button